src/UI/EmailHTMLTooltip.js

```diff
@@ -24,6 +24,7 @@ ${this.options.css}
             <span class="tooltip__button--email__secondary-text">Blocks email trackers and hides your address</span>
         </button>
     </div>
+    <div class="tooltip--email__caret"></div>
```

Comment:
The caret used to be a pseudo element, but now that we're positioning them independently, I created a separate element.